src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -135,9 +133,10 @@ public boolean hasAsset(Asset asset) {
      */
     public void editAsset(Asset target, Asset editedAsset) {
         requireNonNull(target);
+        requireNonNull(editedAsset);
```

Comment:
Would it be better to shorten this to 1 line using the varargs signature `requireAllNonNull(Object...)`?

Reply:
Moved it to Assets#edit(), using the 1 line requireAllNonNull().